src/components/Test/Test.js

```diff
@@ -0,0 +1,26 @@
+import { products } from '../../utils/config';
+
+function Test() {
```

Comment:
oo @kvnloughead looks like you accidentally pushed this up. we shouldn't introduce this to the main branch if we don't need to, ya?

Reply:
Oh, you are correct.

Reply:
Yes, it looks like there is a 'merge and squash' option on github